README.md

```diff
@@ -521,6 +521,8 @@ ANSWERS.addComponent('UniversalResults', {
       renderItem: function(data) {},
       // Optional, override the handlebars template for each item in this vertical
       itemTemplate: `my item {{name}}`,
+      // DEPCREATED, please use viewMoreLabel instead. The text for the view more link. Has the same effect as viewMoreLabel, and is out-prioritized by viewMoreLabel. Defaults to 'View More'.
```

Comment:
typo `DEPCREATED`

could we word somewhere along the lines of the value of `viewMoreLabel` takes precedence over `viewAllText`

Reply:
Does 
>Has the same effect as viewMoreLabel, and is out-prioritized by viewMoreLabel.
work?

Reply:
updated to the wording we agreed on